src/ergtable.h

```diff
@@ -75,6 +75,13 @@ class ergTable : public QObject {
 
             qDebug() << "interpolation" << lowerPoint.wattage + (upperPoint.wattage - lowerPoint.wattage) * (cadenceRatio + resistanceRatio) / 2;
 
+            if(qIsNaN(cadenceRatio) || qIsInf(cadenceRatio)) {
```

Comment:
Can this approach be trusted on every platform? I'd prefer to just not divide by 0 in the first place.